### PR TITLE
Uml 3483 sign into service content

### PR DESCRIPTION
**Pull request by Lbagg1**
# Purpose
On new login page advise customers that by logging in with this service they are accepting the terms of use. - link to https://use-lasting-power-of-attorney.service.gov.uk/terms-of-use 

CONTENT

As a result of looking at this ticket, some other changes to this page are to be made, as per screenshot attached:

The content that signposts the user to the Make an LPA service is to appear above the page header and as part of an information banner.

New content, for the terms of use, is to appear above the ‘Continue’ button, as part of an inset component. The content is:

Terms of use reference: ‘By continuing, you are agreeing to the terms of the Use a lasting power of attorney service.’ (the link here is to go to https://use-lasting-power-of-attorney.service.gov.uk/terms-of-use )

Welsh: 'Ewch i’r Gwasanaeth gwneud atwrneiaeth arhosol  i wneud LPA ar-lein, neu i barhau gyda chais am LPA rydych eisoes wedi’i gychwyn.’ (the underlined Welsh content is the link to the terms)

Acceptance criteria

new content as displayed on attachment

behind onelogin flag

Fixes UML-3483

## Checklist

* [ ] I have performed a self-review of my own code
* [ ] I have added relevant logging with appropriate levels to my code
* [ ] I have added tests to prove my work
* [ ] I have added welsh translation tags and updated translation files
* [ ] The product team have tested these changes

### CODE REVIEW

**Review by allenannom — APPROVED** [review]

File: service-front/app/languages/cy/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -6731,6 +6731,29 @@ msgstr ""
...
+#| "If you have any problems signing in to this service, you can <a "
+#| "href=https://home.account.gov.uk/contact-gov-uk-one-login?lng=en "
+#| "class=\"govuk-link\">contact the GOV.UK One Login team</a> to get help."
+msgid ""
+"If you have any problems signing in to this service, you can <a href=\"https://"
+"home.account.gov.uk/contact-gov-uk-one-login?lng=en\" class=\"govuk-"
+"link\">contact the GOV.UK One Login team</a> to get help."
+msgstr ""
+"Os ydych yn cael unrhyw broblemau wrth fewngofnodi i’r gwasanaeth hwn, gallwch "
+"<a href=https://home.account.gov.uk/contact-gov-uk-one-login?lng=cy "
```
Comment (cooperaj) [inline service-front/app/languages/cy/LC_MESSAGES/messages.po:6754]: missing some quotes on the url here.

File: service-front/app/src/Actor/templates/actor/one-login.html.twig [hunk 2]
```diff
@@ -40,6 +53,17 @@
                             If you have a Use a lasting power of attorney account from before 1 January 2024 and you use the same email address for GOV.UK One Login, any LPAs you’ve previously added will automatically appear.
                         {% endtrans %}
                     </p>
+
+                    <div class="govuk-inset-text">
+                        {% trans with {
+                            '%link_en%': 'https://use-lasting-power-of-attorney.service.gov.uk/terms-of-use',
```
Comment (cooperaj) [inline service-front/app/src/Actor/templates/actor/one-login.html.twig:59]: For relative links it's possible to use our routing code to generate this for you. Bonus is that it handles the welsh version.

https://github.com/ministryofjustice/opg-use-an-lpa/blob/26b327b0961b711560368be4b8bb2a3d5817bc8b/service-front/app/src/Actor/templates/actor/create-account.html.twig#L58

**Review by cooperaj — CHANGES_REQUESTED** [review]
Couple of small tweaks needed.

File: service-front/app/languages/cy/LC_MESSAGES/messages.po [hunk 3]
```diff
@@ -4069,7 +4069,7 @@ msgid ""
 "link\">contact the GOV.UK One Login team</a> to get help."
 msgstr ""
 "Os ydych yn cael unrhyw broblemau wrth fewngofnodi i’r gwasanaeth hwn, gallwch "
-"<a href=https://home.account.gov.uk/contact-gov-uk-one-login?lng=cy "
+"<a href=“https://home.account.gov.uk/contact-gov-uk-one-login?lng=cy” "
```
Comment (cooperaj) [inline service-front/app/languages/cy/LC_MESSAGES/messages.po:4072]: These will need to be escaped ASCII quotes 

```suggestion
"<a href=\"https://home.account.gov.uk/contact-gov-uk-one-login?lng=cy\" "
```

PoEdit definitely not doing the right thing here and it's annoying. Typically it's a manual fix in PHPStorm after using PoEdit if any of the new Welsh has HTML in it, like a link.

File: service-front/app/languages/cy/LC_MESSAGES/messages.po [hunk 4]
```diff
@@ -189,51 +189,54 @@ msgstr ""
...
-"attorney\">LPA</abbr>."
+"Give %organisation% this access code. They should go to <a href=\"%link_en%\" "
+"class=\"govuk-link\">www.gov.uk/view-lpa</a> to enter the access code and see "
+"a summary of the <abbr title=\"lasting power of attorney\">LPA</abbr>. "
+"%organisation% may still ask to see the paper LPA."
 msgstr ""
-"Rhowch y cod mynediad hwn i %organisation%. Dylent fynd i www.gov.uk/view-lpa "
-"i roi’r cod mynediad a gweld crynodeb o’r atwrneiaeth."
+"Rhowch y cod mynediad hwn i %organisation%. Dylent fynd i <a href=%link_cy% "
+"class=“govuk-link”>www.gov.uk/gweld-atwrneiaeth-arhosol</a> i roi’r cod "
```
Comment (cooperaj) [inline service-front/app/languages/cy/LC_MESSAGES/messages.po:200]: Seems there are a number of new changes in this file. It's necessary to be quite careful with how these quotes gets introduced as they can stop the translations working entirely.
```suggestion
"class=\"govuk-link\">www.gov.uk/gweld-atwrneiaeth-arhosol</a> i roi’r cod "
```

File: service-front/app/languages/cy/LC_MESSAGES/messages.po [hunk 5]
```diff
@@ -4058,14 +4033,27 @@ msgstr ""
 "bydd unrhyw atwrneiaeth arhosol rydych chi wedi’i hychwanegu’n flaenorol yn "
 "ymddangos yn awtomatig."
 
-#: src/Actor/templates/actor/one-login.html.twig:56
+#: src/Actor/templates/actor/one-login.html.twig:58
+msgid ""
+"By continuing, you are agreeing to the <a href=\"%link%\" class=\"govuk-"
+"link\">terms of the Use a lasting power of attorney service.</a>"
+msgstr ""
+"Trwy barhau, rydych yn cytuno i <a href=%link% class=\"govuk-link\"> delerau’r "
```
Comment (cooperaj) [inline service-front/app/languages/cy/LC_MESSAGES/messages.po:4041]:
```suggestion
"Trwy barhau, rydych yn cytuno i <a href=\"%link%\" class=\"govuk-link\"> delerau’r "
```

File: service-front/app/languages/cy/LC_MESSAGES/messages.po [hunk 6]
```diff
@@ -6677,59 +6653,96 @@ msgctxt "flashMessage"
...
-"Give %organisation% this access code. They should go to <a href=\"%link_en%\" "
-"class=\"govuk-link\">www.gov.uk/view-lpa</a> to enter the access code and see "
-"a summary of the <abbr title=\"lasting power of attorney\">LPA</abbr>. "
-"%organisation% may still ask to see the paper LPA."
-msgstr ""
-"Rhowch y cod mynediad hwn i %organisation%. Dylent fynd i <a href=%link_cy% "
-"class=“govuk-link”>www.gov.uk/gweld-atwrneiaeth-arhosol</a> i roi’r cod "
-"mynediad a gweld crynodeb o’r atwrneiaeth. Efallai y bydd %organisation% yn "
-"dal i ofyn am weld yr LPA bapur."
+#~ msgid ""
```
Comment (cooperaj) [inline service-front/app/languages/cy/LC_MESSAGES/messages.po:6656]: Looks like a fair number of changes to the viewercode side of things. Probably nothing but could you please verify the pages that have (or did have) this content on are still functioning. That goes for all the viewercode items in this diff.

File: service-front/app/src/Actor/templates/actor/one-login.html.twig [hunk 7]
```diff
@@ -40,6 +53,16 @@
                             If you have a Use a lasting power of attorney account from before 1 January 2024 and you use the same email address for GOV.UK One Login, any LPAs you’ve previously added will automatically appear.
                         {% endtrans %}
                     </p>
+
+                    <div class="govuk-inset-text">
+                        {% trans with {
+                            '%link%': path('terms-of-use')
```
Comment (cooperaj) [inline service-front/app/src/Actor/templates/actor/one-login.html.twig:59]: Perfect.

File: service-front/app/src/Actor/templates/actor/one-login.html.twig [hunk 8]
```diff
@@ -12,13 +12,26 @@
...
+                    <div class="moj-banner" role="region" aria-label="information">
+                        <svg class="moj-banner__icon" fill="currentColor" role="presentation" focusable="false"
+                             xmlns="http://www.w3.org/2000/svg" viewBox="0 0 25 25" height="25" width="25">
+                            <path d="M13.7,18.5h-2.4v-2.4h2.4V18.5z M12.5,13.7c-0.7,0-1.2-0.5-1.2-1.2V7.7c0-0.7,0.5-1.2,1.2-1.2s1.2,0.5,1.2,1.2v4.8
+                                C13.7,13.2,13.2,13.7,12.5,13.7z M12.5,0.5c-6.6,0-12,5.4-12,12s5.4,12,12,12s12-5.4,12-12S19.1,0.5,12.5,0.5z" />
+                        </svg>
+                        <div class="moj-banner__message">
+                            <h2 class="govuk-body">
+                                {% trans with {
+                                    '%link%': 'https://www.gov.uk/power-of-attorney/make-lasting-power'
```
Comment (cooperaj) [inline service-front/app/src/Actor/templates/actor/one-login.html.twig:24]: I wonder if there is an equivalent welsh version of this page?

**Comment by cooperaj** [comment]
I've got a series of find-and-replace operations I typically carry out on the PO file when I'm done in Poedit. I'll get them documented (or make a script) so that the double quote thing can be easily watched in the future.

The issue is that you don't just want to replace all of them (or the single quote equivalents) as they might legitimately be being used in the text. It only bites us when they end up in the html (if a translation has any)

File: service-front/app/src/Actor/templates/actor/one-login.html.twig (hunk 8 above)
Comment (Lbagg1) [reply service-front/app/src/Actor/templates/actor/one-login.html.twig:24]: there isn't an Welsh version , I have checked with Sarah.

**Review by cooperaj — APPROVED** [review]

**Pull request merged**